packages/netlify-cms-core/src/actions/__tests__/entries.spec.js

```diff
@@ -108,61 +112,35 @@ describe('entries', () => {
     });
   });
 
-  describe('discardDraft', () => {

```

Comment:
I moved `mediaFiles` property from `entryDraft. mediaFiles` to `entryDraft.entry.mediaFiles` so now they are synced when `entryDraft.entry` is synced